docs/pr-guide.md

```diff
@@ -25,21 +25,55 @@ Anyone with write access can merge a pull request manually or by setting the [au
+  * Perhaps you see the same failure in CI results for unrelated active PR's.
+  * It's a known issue listed in our [big tracking issue](https://github.com/dotnet/runtime/issues/702) or tagged `blocking-clean-ci` [(query here)](https://github.com/dotnet/runtime/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Ablocking-clean-ci+)
+  * Its otherwise beyond doubt that your code changes could not have caused this.
+  * If the tests pass on rerun, that may suggest it's not related.
+* In this situation, you want to re-run but not necessarily rebase on master.
+  * To rerun just the failed leg(s):
+    * Click on any leg. Navigate through the Azure DevOps UI, find the "..." button and choose "Retry failed legs"
+    * Or, on the GitHub Checks tab choose "re-run failed checks". This will not rebase your change.
+  * To rerun all validation:
+    * Add a comment `/azp run runtime`
+    * Or, click on "re-run all checks" in the GitHub Checks tab
+    * Or, simply close and reopen the PR.
+* If you have established that it is an unrelated failure, please ensure we have an active issue for it. See the [unrelated failure](#unrelated-failure) section below.
+* Whoever merges the PR should be satisfied that the failure is unrelated, is not introduced by the change, and that we are appropriately tracking it.
+
+### Option 3: The state of the master branch HEAD is bad.
+
+* This is the very rare case where there was a build break in master, and you got unlucky. Hopefully the break has been fixed, and you want CI to rebase your change and rerun validation.
+* To do this, add a comment `/azp run runtime`, or click on "re-run all checks" in the GitHub Checks tab, or just close and reopen the PR.
+
+### Additional information:
+  * You can list the available pipelines by adding a comment like `/azp list` or get the available commands by adding a comment like `azp help`.
+  * Reach out to the infrastructure team for assistance on [Teams channel](https://teams.microsoft.com/l/channel/19%3ab27b36ecd10a46398da76b02f0411de7%40thread.skype/Infrastructure?groupId=014ca51d-be57-47fa-9628-a15efcc3c376&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) (for corpnet users) or on [Gitter](https://gitter.im/dotnet/community) in other cases.
+
+## What to do if you determine the failure is unrelated
+
+If you have determined the failure is definitely not caused by changes in your PR, please do this:
+
+* Search for an [existing issue](https://github.com/dotnet/runtime/issues). Usually the test method's or the test assembly's name (in case of a crash) are good search parameters.
+  * If there's an existing issue, add a comment with a) the link to the build, b) the affected configuration (ie `netcoreapp-Windows_NT-Release-x64-Windows.81.Amd64.Open`) and c) the Error message and Stack trace. This is necessary as retention policies are in place that recycle _old_ builds.
```

Comment:
Sometimes there are crashes, and there is no stack trace, so could you mention including the link to the crashdump and/or console log?